work_queue/src/work_queue.c

```diff
@@ -950,11 +952,15 @@ high throughput and low latency.
 static work_queue_result_code_t get_file_or_directory( struct work_queue *q, struct work_queue_worker *w, struct work_queue_task *t, const char *remote_name, const char *local_name, int64_t * total_bytes)
 {
 	// Remember the length of the specified remote path so it can be chopped from the result.
-	int remote_name_len = strlen(remote_name);
 
 	// Send the name of the file/dir name to fetch
 	debug(D_WQ, "%s (%s) sending back %s to %s", w->hostname, w->addrport, remote_name, local_name);
-	send_worker_msg(q,w, "get %s 1\n",remote_name);
+	if(disable_cache) {
```

Comment:
Yikes -- this is the wrong way to go about it.
Even when files are not cached between jobs, they still need to into the cache directory,
so that the worker can name them into the right place, and then account for the storage.
The master will still delete the files when the task is done.